test/suites/dev/moonbase/test-fees/test-fee-multiplier-max.ts

```diff
@@ -84,7 +84,7 @@ describeSuite({
         // ~/4 to compensate for the ref time XCM fee changes
         // Previous value: 449_284_776_265_723_667_008n
         // Previous value: 119_241_298_837_127_813_277n
-        expect(initialBalance - afterBalance).to.equal(72_611_605_800_552_813_277n);
+        expect(initialBalance - afterBalance).to.equal(146_015_659_550_552_813_277n);
```

Comment:
The reason for this big difference is an under-estimation introduced here: https://github.com/moonbeam-foundation/moonbeam/pull/3217